test/test_helper.rb

```diff
@@ -32,7 +32,7 @@ module ActiveSupport
   class TestCase
     include FactoryBot::Syntax::Methods
 
-    parallelize(workers: :number_of_processors)
+    # parallelize(workers: :number_of_processors)
```

Comment:
i did because when i ran the systems test it spun up 4x servers and just killed my computer (i have a lot of stuff open to do all my projects). this then led the tests to have a 50% failure rate because they kept timing out (the socket).